From 5d7ed72ffc8cf3ae761709c74cdf33276ffa8949 Mon Sep 17 00:00:00 2001
From: William Roberts <william.c.roberts@intel.com>
Date: Mon, 21 Aug 2017 10:48:11 -0700
Subject: [PATCH 4/4] travis: use master branch for tss

Switch abrmd over to use the master branch of the tss.

Signed-off-by: William Roberts <william.c.roberts@intel.com>
---
 .travis.yml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.travis.yml b/.travis.yml
index e412698..cfbe764 100644
--- a/.travis.yml
+++ b/.travis.yml
@@ -49,7 +49,7 @@ install:
   - cmake ../ -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release
   - make -j$(nproc) install
   - popd
-  - git clone -b 1.x --single-branch https://github.com/01org/tpm2-tss.git
+  - git clone -b master --single-branch https://github.com/01org/tpm2-tss.git
   - pushd tpm2-tss
   - ./bootstrap
   - ./configure --prefix=${PREFIX}
-- 
2.7.4

